lib/paper_trail/frameworks/active_record.rb

```diff
@@ -2,3 +2,7 @@
 # since otherwise the model(s) will get loaded in via the `Rails::Engine`.
 require "paper_trail/frameworks/active_record/models/paper_trail/version_association"
 require "paper_trail/frameworks/active_record/models/paper_trail/version"
+
+ActiveSupport.on_load(:active_record) do
```

Comment:
Oh I see now that you also added a similar block inside of the `Rails::Engine`. Did this have issues too related to thread safety?